src/views/Details/NFTAssetDetails.vue

```diff
@@ -0,0 +1,394 @@
+<template>
+  <div class="nft-asset">
+    <div class="nft-asset--container">
+      <NavBar showBack="true" :backPath="'/wallet/nfts'" backLabel=""></NavBar>
+      <div class="nft-action-buttons">
+        <div class="star" :style="showFullscreen ? { top: '120px' } : { top: '40px' }">
```

Comment:
This code piece seem similar (or maybe its the same) as the one in `NFTAsset.vue`

Reply:
No, this is different.